ldi-nifi/ldi-nifi-processors/ldes-client-processor/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/processors/LdesClientProcessor.java

```diff
@@ -126,7 +127,7 @@ private Retry getRetry(final ProcessContext context) {
 
 	private RequestExecutor getRequestExecutor(final ProcessContext context) {
 		return switch (getAuthorizationStrategy(context)) {
-			case NO_AUTH -> requestExecutorFactory.createNoAuthExecutor();
+			case NO_AUTH -> requestExecutorFactory.createNoAuthExecutor(new ArrayList<>());
```

Comment:
Why is this empty list suddenly added as a param? Doesn't the original line do the same anymore?

Reply:
Same story as this morning, I did something else first when this made sense, reverted the changes and forgot to revert this :smile_cat: